invoice/src/main/java/org/killbill/billing/invoice/dao/InvoiceDao.java

```diff
@@ -149,4 +149,32 @@ InvoicePaymentModelDao createRefund(UUID paymentId, BigDecimal amount, boolean i
      * @throws InvoiceApiException
      */
     void changeInvoiceStatus(UUID invoiceId, InvoiceStatus newState, InternalCallContext context) throws InvoiceApiException;
+
+    /**
+     * Save parent/child invoice relationship
+     *
+     * @param invoiceRelation the invoice relation object
+     * @param context the tenant context
+     * @throws InvoiceApiException
```

Comment:
mmm... I'm not sure what do you mean. I'm using jdk1.8 profile and I don't see any error on javadocs. I already created them by "/** + intro"